caikit/core/data_model/dataobject.py

```diff
@@ -145,7 +148,19 @@ class MyDataObject(DataObjectBase):
         directly, the metaclass that links protobufs to the class will be called
         before this decorator can auto-gen the protobufs class.
 
-    Args:
+    The `dataobject` decorator will not provide tools with enough information to perform type completion
```

Comment:
Hmm, sure, will do, but I think the default column width seems to be 100, per `.pylintrc`?  (I had my PyCharm set  at 120)

Also, should we have
```
[tool.black]
line-length = 100
include = '\.pyi?$'
exclude = '''
/(
    \.git
  | \.hg
  | \.mypy_cache
  | \.tox
  | \.venv
)/
'''
```

in `pyproject.toml`, so that black formatting will work consistently?